docs/source/dev.md.inc

```diff
@@ -29,9 +29,10 @@
     1. Emptyroom files that lack cHPI channels will now be processed (for line noise only) instead of raising an error.
     2. cHPI filtering is now performed before movement compensation.
 
-- Fix bug where the ``config.proc`` parameter was not used properly during forward model creation (#1014 by @larsoner)
+- Fix bug where the `config.proc` parameter was not used properly during forward model creation (#1014 by @larsoner)
 - Fix bug where emptyroom recordings containing EEG channels would crash the pipeline during maxwell filtering (#1040 by @drammock)
 - Fix bug where only having mag sensors would crash compute_rank in maxwell filtering (#1061 by @harrisonritz)
+- Drop legacy function `inst.pick_types` in favor of `inst.pick` (#1073 by @PierreGtch)
```

Comment:
FWIW I would probably have put this under "code health" rather than bugfix. But don't change it now, no need to re-run CIs just for that. We can move it in a subsequent PR.